Input/input.css

```diff
@@ -0,0 +1,94 @@
+:root{
+    --white-color:white;
+    --black-color:black;
+    --font-size:18px;
+}
+.code{
+    margin-top:2rem;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    background-color:rgb(139, 185, 179);
+    color:black;
+    font-weight: bold;
+    font-size:19px;
```

Comment:
Could have used rem like the rest. Apart from that everything else looks good and clean.

Reply:
sure